doc/manual/manual.tex

```diff
@@ -2005,13 +2005,13 @@ \subsubsection{Combined formulations}
 (Section~\ref{sec:tala}).
 
 \item
-``boussinesq approximation'': This formulation sets the mass conservation approximation to ``incompressible'', 
+``Boussinesq approximation'': This formulation sets the mass conservation approximation to ``incompressible'', 
```

Comment:
Yes, that should be changed in all files where it occurs (this PR is just a recursive sed find-replace)